packages/next/src/types.ts

```diff
@@ -1,8 +1,8 @@
-/// <reference types="node" />
-/// <reference types="react" />
-/// <reference types="react/experimental" />
-/// <reference types="react-dom" />
-/// <reference types="react-dom/experimental" />
+/// <reference types="node" preserve="true" />
+/// <reference types="react" preserve="true" />
+/// <reference types="react/experimental" preserve="true" />
+/// <reference types="react-dom" preserve="true" />
+/// <reference types="react-dom/experimental" preserve="true" />
```

Comment:
Use preserve true to prevent it removed.

x-ref:

- [simplified-reference-directive-declaration-emit](https://devblogs.microsoft.com/typescript/announcing-typescript-5-5/#simplified-reference-directive-declaration-emit)
- https://github.com/microsoft/TypeScript/pull/57681

Reply:
I was looking for something like this the last time. Nice find 👍🏻 The TS team is not the only team [missing docs](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html) 😄